src/agent/src/random.rs

```diff
@@ -13,7 +13,7 @@ use tracing::instrument;
 
 pub const RNGDEV: &str = "/dev/random";
 pub const RNDADDTOENTCNT: libc::c_int = 0x40045201;
-pub const RNDRESEEDRNG: libc::c_int = 0x5207;
+pub const RNDRESEEDCRNG: libc::c_int = 0x5207;
```

Comment:
Not an expert in kernel nor security writing...
... is CRNG (Cryptographic RNG) same as RNG? Does `/dev/random` device uses a cryptographic algorithm?

Reply:
@wainersm - this is correcting a typo - see:

- https://github.com/torvalds/linux/blob/master/include/uapi/linux/random.h#L39

In fact, it might be a good idea @etrunko if you added a link to the linux source file in a comment to make this clearer.